documentation/source/tutorials/pages/registration_to_template.rst

```diff
@@ -0,0 +1,293 @@
+.. _registration-to-template:
+
+Tutorial 2: Registering T2 images to the PAM50 template
+#######################################################
+
+This tutorial demonstrates a multi-step pipeline to register an anatomical MRI scan to the PAM50 Template. While T2 images are used for this tutorial, each step is applicable to multiple contrast types (T1, T2, T2*).
+
+Prerequisites
+*************
+
+1. Read through the following pages to familiarize yourself with key SCT concepts:
+
+    * :ref:`pam50`: An overview of the PAM50 template's features, as well as context for why the template is used.
+    * :ref:`warping-fields`: Background information on the image transformation format used by the registration process.
+    * :ref:`qc`: Primer for SCT's Quality Control interface. After each step of this tutorial, you will be able to open a QC report that lets you easily evaluate the results of each command.
+
+2. Download and unzip `sct_course_london20.zip <https://osf.io/bze7v/?action=download>`_.
```

Comment:
i prefer it that way, compared to how it is currently done for the first tutorial on cord segmentation: a big ugly warning tells ppl to download the example dataset. We should harmonize.

alternatively, we can have a section at the very beginning about "do this before starting the tutorial"

Reply:
> alternatively, we can have a section at the very beginning about "do this before starting the tutorial"

That was exactly what I was thinking for the "Prerequisites" section -- but you're right, "Before starting this tutorial" is a clearer name for the beginning section. :slightly_smiling_face: